locale/af_ZA/app.po

```diff
@@ -1895,7 +1895,7 @@ msgstr ""
 msgid "Invalid token"
 msgstr ""
 
-msgid "Is {{email_address}} the wrong address for {{type_of_request}} requests to {{public_body_name}}? If so, please contact us using this form:"
```

Comment:
Preserving the existing translations is a good thing to do, but we need to be a bit careful about how we push these to Transifex without destroying translations that we haven't pulled into the repo yet. What was your rollout plan here?

Reply:
Will need to do a manual process of:
1. pull from Transifex - this replace these changes
2. re-apply this commit patch
3. push back up to Transifex